instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -81,155 +82,598 @@ def __init__(
+            observer_type=SumObserver,
+        )
+
+        self.meter.register_observer(
+            callback=self._get_system_disk_merged,
+            name="system.disk.merged",
+            description="System disk merged",
+            unit="1",
+            value_type=int,
+            observer_type=SumObserver,
+        )
+
+        # self.meter.register_observer(
+        #     callback=self._get_system_filesystem_usage,
+        #     name="system.filesystem.usage",
+        #     description="System filesystem usage",
+        #     unit="bytes",
+        #     value_type=int,
+        #     observer_type=UpDownSumObserver,
+        # )
+
+        # self.meter.register_observer(
+        #     callback=self._get_system_filesystem_utilization,
+        #     name="system.filesystem.utilization",
+        #     description="System filesystem utilization",
+        #     unit="1",
+        #     value_type=float,
+        #     observer_type=UpDownSumObserver,
+        # )
+
+        self.meter.register_observer(
+            callback=self._get_system_network_dropped_packets,
+            name="system.network.dropped_packets",
+            description="System network dropped_packets",
+            unit="packets",
+            value_type=int,
+            observer_type=SumObserver,
+        )
+
+        self.meter.register_observer(
+            callback=self._get_system_network_packets,
+            name="system.network.packets",
+            description="System network packets",
+            unit="packets",
+            value_type=int,
+            observer_type=SumObserver,
+        )
+
+        self.meter.register_observer(
+            callback=self._get_system_network_errors,
+            name="system.network.errors",
+            description="System network errors",
+            unit="errors",
+            value_type=int,
+            observer_type=SumObserver,
         )
 
         self.meter.register_observer(
-            callback=self._get_network_bytes,
-            name="system.net.bytes",
-            description="System network bytes",
+            callback=self._get_system_network_io,
+            name="system.network.io",
+            description="System network io",
             unit="bytes",
             value_type=int,
-            observer_type=ValueObserver,
+            observer_type=SumObserver,
+        )
+
+        self.meter.register_observer(
+            callback=self._get_system_network_connections,
+            name="system.network.connections",
+            description="System network connections",
+            unit="connections",
+            value_type=int,
+            observer_type=UpDownSumObserver,
         )
 
         self.meter.register_observer(
-            callback=self._get_runtime_memory,
-            name="runtime.python.mem",
-            description="Runtime memory",
+            callback=self._get_runtime_cpython_memory,
+            name="runtime.cpython.memory",
```

Comment:
Maybe format in the python implementation? https://docs.python.org/3/library/platform.html#platform.python_implementation

Reply:
Fixed